### ISSUE TITLE
/Carpet isn't coming up as an available command

### ISSUE THREAD

**Issue by jwattsy** [gnembon/fabric-carpet#102]
I have the carpet mod in the Minecraft mod folder for my 3rd party run server, however none of the available commands show up for the carpet mod, they are all still the default Minecraft commands

**Comment by DeadlyMC** [comment]
Are u sure the mod is running? Also you need cheats enabled (or OP on a server) for the command to be available

**Comment by gnembon** [comment]
in the F3 menu up top it should say fabric.... carpet.....

to be able to configure carpet, you need to have OP permissions

**Comment by jwattsy** [comment]
![Carpet](https://user-images.githubusercontent.com/57697458/68791458-5b0fd480-0617-11ea-8be0-ef23d8542f13.PNG)

Figured it would be able to show you what it says, I tried it in a single player world and the carpet command comes up.
The screenshot is of the survival multiplayer world I am trying to use it in.

**Comment by jwattsy** [comment]
Not sure if this matters but these are the mods I currently am using
![1](https://user-images.githubusercontent.com/57697458/68813449-d6877b00-0643-11ea-8bca-25551f7f3e0c.PNG)

**Comment by DeadlyMC** [comment]
Do you have the mod on the server ? Having it just on your client and connecting to a server won't work

**Comment by jwattsy** [comment]
I don't think so, I have access to the server folders but there is no mod folder. Would I need to download the fabric launcher and the carpet mod into the server files? (FYI using the vanilla launcher, not multiMC)

**Comment by DeadlyMC** [comment]
You need to setup a fabric server and add the carpet...jar to the mods folder (Mostly it won't be present...u can create one named `mods`) in your server directory.
You can follow this guide to setup a fabric server: https://fabricmc.net/wiki/install

**Comment by AigilasDuckii** [comment]
Hi, I kinda have the same issue here I think and I don´t understand what to do.

I own a Paper server, I have the carpet mod installed locally (just tested in singleplayer) but I don´t understand how to enable it on the server. The "create a folder named ""mods" and put the .jar file in it" method didn´t work.
Thanks in advance

**Comment by DeadlyMC** [comment]
Carpet requires fabric mod loader. That's why it doesn't run on Paper

**Comment by AigilasDuckii** [comment]
Any suggestions on what I can do? I´d hate to restart the world we´ve made so far

**Comment by DeadlyMC** [comment]
My best bet would be to ask on the paper discord if its possible to load a paper world in vanilla

**Comment by benso87** [comment]
Sorry to bring this back up, but I'm having the same problem originally mentioned. I'm the server owner, and we were using carpet just fine on 1.14.4. Once 1.15 came out, I updated the server to fabric 1.15 and replaced the carpet and carpet-extra jar files with the updated ones from here (also did all of that on my computer as well). I replaced them all again today to get to 1.15.1. Attached is a screenshot of what the version in the F3 menu shows in both situations.

![minecraft version](https://user-images.githubusercontent.com/59041777/71147318-149d2f00-21ee-11ea-835f-5780502e6aff.png)

The commands work in single player, but not on the server. My local mods folder and the mods folder on the server are identical. What am I missing?
